### DISCUSSION TITLE
I can't seem to schedule Cron jobs if any value is two digits

### DISCUSSION THREAD

**Discussion by conleyc86** [appwrite/appwrite#6038] [Q&A] [answered]
Am I doing something wrong or do I need to find a workaround? New to Appwrite but have written plenty of cron expressions before.

Here's a screenshot of the issue...
![image](https://github.com/appwrite/appwrite/assets/24814715/cce0c4dd-9b23-4a30-9661-278281297976)

**Comment by mauricev** [comment, marked as answer]
Might this be the answer, https://discord.com/channels/564160730845151244/1150301893889896498

**Reply by conleyc86** [reply]
I marked this for future reference - thanks!

**Comment by stnguyen90** [comment]
This issue is in the process of being fixed: https://github.com/appwrite/appwrite/issues/5719

**Reply by conleyc86** [reply]
Awesome - thank you!
